docs/V7-Migration-Guide.md

```diff
@@ -355,21 +355,9 @@ Latest example can be found on [widgets.dojo.io/#widget/progress/overview](https
 
 ### radio
 
-#### Property changes
```

Comment:
why did you remove all these headers? I think label after needs to go in `changed properties` ?